sdk/python/tests/expediagroup/test_milvus_online_store.py

```diff
@@ -1,37 +1,346 @@
+            Field(
+                name="feature2",
+                dtype=Int64,
+                tags={"is_primary": "True", "description": "int64"},
+            ),
+        ]
+
+        # Creating a common schema for collection to directly add to Milvus
+        schema = CollectionSchema(
+            fields=[
+                FieldSchema(
+                    "int64", DataType.INT64, description="int64", is_primary=True
+                ),
+                FieldSchema(
+                    "float_vector", DataType.FLOAT_VECTOR, is_primary=False, dim=128
+                ),
+            ]
+        )
+
+        # Here we want to open and add a collection using pymilvus directly and close the connection
+        with PymilvusConnectionContext():
+            Collection(name=self.collection_to_write, schema=schema)
+            assert utility.has_collection(self.collection_to_write) is True
+
+        MilvusOnlineStore().update(
+            config=repo_config,
+            tables_to_delete=[
+                VectorFeatureView(
+                    name=self.collection_to_write,
+                    schema=schema_vector,
+                    source=SOURCE,
+                    vector_field=VECTOR_FIELD,
+                    dimensions=DIMENSIONS,
+                    index_algorithm=INDEX_ALGO,
+                )
+            ],
+            tables_to_keep=[],
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
+
+        # Opening and closing the connection and checking if the collection is actually deleted.
+        with PymilvusConnectionContext():
+            assert utility.has_collection(self.collection_to_write) is False
+
+    def test_milvus_update_delete_unavailable_collection(
+        self, repo_config, caplog, milvus_online_setup
+    ):
+        schema_vector = [
+            Field(
+                name="feature1",
+                dtype=Array(Float32),
+                tags={
+                    "is_primary": "False",
+                    "description": "float32",
+                    "dimension": "128",
+                },
+            ),
+            Field(
+                name="feature2",
+                dtype=Int64,
+                tags={"is_primary": "True", "description": "int64"},
+            ),
+        ]
+
+        MilvusOnlineStore().update(
+            config=repo_config,
+            tables_to_delete=[
+                VectorFeatureView(
+                    name="abc",
+                    schema=schema_vector,
+                    source=SOURCE,
+                    vector_field=VECTOR_FIELD,
+                    dimensions=DIMENSIONS,
+                    index_algorithm=INDEX_ALGO,
+                )
+            ],
+            tables_to_keep=[],
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
+
+        assert "Collection abc does not exist or is already deleted." in caplog.text
```

Comment:
minor: I'd rather assert against Milvus not having any collection. The update() method should really have a return value, but that is an issue in Feast itself